client/src/api/config.ts

```diff
@@ -1,4 +1,7 @@
 import { execOnHost, isWindows, throwErrorAsString } from './utils';
+import {createDockerDesktopClient} from "@docker/extension-api-client";
```

Comment:
Missing white-space after "(" and before "}" - so that this code line looks the same as the code lines above it.
The same goes for this line in ```Scan.tsx``` and in a few other files.